packages-ts/gauntlet-terra-cw-plus/src/commands/multisig.ts

```diff
@@ -91,15 +94,38 @@ export const wrapCommand = (command) => {
       }
     }
 
+    parseVotingPeriod(votingPeriod: string): number {
+      try {
+        Number(votingPeriod)
+      } catch {
+        throw new Error(
+          `Invalid --votingPeriod, must be a duration in seconds (default: ${DEFAULT_VOTING_PERIOD_IN_SECS}, max: ${MAX_VOTING_PERIOD_IN_SECS})`,
+        )
+      }
+      const n = Number(votingPeriod)
+      if (n < 0 || n > MAX_VOTING_PERIOD_IN_SECS) {
+        throw new Error(
+          `--votingPeriod out of range (default: ${DEFAULT_VOTING_PERIOD_IN_SECS}, max: ${MAX_VOTING_PERIOD_IN_SECS})`,
+        )
+      }
+      return n
+    }
+
     makeProposeTransaction: ProposalAction = async (signer, _, message) => {
+      // default voting period of 24 hours, if unspecified
+      const votingPeriod: number = this.flags.votingPeriod
+        ? this.parseVotingPeriod(this.flags.votingPeriod)
+        : DEFAULT_VOTING_PERIOD_IN_SECS
+
       logger.info('Generating data for creating new multisig proposal')
+      const expiresAt = (Date.now() + votingPeriod * 1000) * time.Millisecond
```

Comment:
Please double check this, as if `votingPeriod` is in seconds then * 1000 gives us milliseconds. 

What should the `expiresAt` unit be (currently it's microseconds)?

Reply:
Uf, it makes sense now. Thx!

If nanosecond is defined as 1, and `Millisecond = 1000 * Microsecond` that makes sense.